README.md

```diff
@@ -60,7 +63,35 @@ tox
 
 ## Runtime Environment
 
-Install (from sources) system-wide:
+### Install pre-requirements
+```shell
+sudo yum -y install epel-release
+sudo yum -y install \
+    libibverbs \
+    NetworkManager \
+    NetworkManager-libnm \
+    NetworkManager-ovs \
+    openvswitch \
+    python2-pyyaml \
+    python2-six \
+    python-gobject-base \
+    python-jsonschema \
+    python-setuptools \
+    python-ipaddress
+sudo echo -e "[device]\nmatch-device=*\nmanaged=0\n" >> \
+    /etc/NetworkManager/conf.d/97-docker-build.conf
+sudo echo -e "[main]\nno-auto-default=*\n" >> \
+    /etc/NetworkManager/conf.d/97-docker-build.conf
+```
+
+### Install nmstate from pip
+```shell
+sudo yum -y install python2-pip
+sudo pip install --upgrade nmstate
```

Comment:
Is this really something that should be recommended? It does not seem to be good advice. IMHO it is better to recommend to install it for the user only.

Reply:
The `--upgrade` command also seems to be dangerous as it seems to make pip upgrade all the requirements as well. IMHO `pip uninstall nmstate; pip install nmstate` seems to be a better recommendation.

Reply:
Ok, dropping the upgrade and replacing it with the uninstall-install.
nmstate in my view is a host level tool, no different from NM. I see no point installing it under a user.

Reply:
For host level installs it is IMHO better to (recommend to) install the RPM package instead of using pip.